### PR TITLE
Merged Alertbox into Alert

### LINKED ISSUE #19129: Merge Alert & AlertBox
we have two components that are 99% identical. 

We should merge them (add features from AlertBox to Alert). And only use Alert (from @grafana/ui).

### PR DESCRIPTION
**Pull request by srid12**
Fixes #19129
 Merged Alert and AlertBox

### CODE REVIEW

**Comment by torkelo** [comment]
Thanks for contributing!

File: packages/grafana-ui/src/components/Alert/Alert.tsx [hunk 1]
```diff
@@ -1,21 +1,49 @@
...
+  Warning = 'warning',
+  Error = 'error',
+  Info = 'info',
+}
+
+interface AlertProps {
   title: string;
   button?: {
-    text: string;
+    text?: string;
```
Comment (torkelo) [inline packages/grafana-ui/src/components/Alert/Alert.tsx]: I would change this nested object form to buttonText?, onButtonClick?, onRemove?  if onRemove is specified it should be rendered like this
image

**Review by torkelo — COMMENTED** [review]
One comment about the notification alerts, they need to look the same with X icon.

File: packages/grafana-ui/src/components/Alert/Alert.tsx (hunk 1 above)
Comment (srid12) [reply packages/grafana-ui/src/components/Alert/Alert.tsx]: Thanks torkleo. If both buttonText and onRemove is specified, shall we give preference to onRemove and show the remove icon only or show icon along with buttonText?

Comment (torkelo) [reply packages/grafana-ui/src/components/Alert/Alert.tsx]: yes!

Comment (srid12) [reply packages/grafana-ui/src/components/Alert/Alert.tsx]: I have updated the PR. Please take a look

**Review by torkelo — APPROVED** [review]

**Pull request merged**